webapps-schema/src/main/resources/schema/elasticsearch/create/index/camunda-authorized-clients.json

```diff
@@ -0,0 +1,55 @@
+{
+  "aliases": {},
+  "mappings": {
+    "dynamic": "true",
```

Comment:
Any particular reason why we are specifying dynamic schema here and not `"dynamic": "strict"`? I know it is like that for the session index, but if I recall correctly, that's becasue we are indexing a dynamic map of session attributes there. Here we have a static set of properties.

Reply:
That is fixed. However, I found another issue with schema migrator.

When there is `"index": true` value on the property, migrator complains that it needs to have a migration. I will report the issue later.